app/code/Magento/CatalogGraphQl/Model/Resolver/Product/ProductImage/Url.php

```diff
@@ -71,7 +79,15 @@ private function getImageUrl(string $imageType, ?string $imagePath): string
         $image = $this->productImageFactory->create();
         $image->setDestinationSubdir($imageType)
             ->setBaseFile($imagePath);
-        $imageUrl = $image->getUrl();
+
+        $imageUrl = $image->isBaseFilePlaceholder()
```

Comment:
@naydav could we get someone to test this solution please?

Reply:
We already have test, need just unskip it and check